docs/tokenomics.md

```diff
@@ -5,12 +5,45 @@ title: HDX Tokenomics
 
 import useBaseUrl from '@docusaurus/useBaseUrl'; 
 
-## Supply Breakdown
+## Purpose
+
+The HDX token is a governance token that allows staked token holders to decide the future of the protocol. Our mission with the HydraDX DAO is to distribute all decision-making to create a trustless liquidity protocol built around community-growth and self-sustainability. 
+
+To have a vote in the HydraDX DAO, and to contribute to the determination of any of the topics raised by community members, one must hold the HDX governance token. For more specifically on the governance process, please read our [Democracy](https://docs.hydradx.io/democracy_intro) documentation.
+
+HDX will initially be used for the following:
+
+- Setting the base network swap fee (the user may change this to any asset)
+- Voting on protocol upgrades
+- Voting on topics raised by community members (inclusive of allocation of Protocol-Owned Liquidity)
+
+## HDX Token Allocation
+
+Upon the launch of the HydraDX DAO, 10 billion HDX tokens were minted. However through the [governance-approved supply reduction](https://hydradx.subsquare.io/democracy/referendum/7), the total supply has been reduced to 6.5 billion HDX tokens.
```

Comment:
they weren't minted - the 10 billion was defined as the maximum supply before being reduced to a maximum supply of 6.5B. Again, these have not been minted, so we should be clear and use the correct terms of "defined maximum" instead of talking about minting